webservice/model/log.go

```diff
@@ -78,47 +83,51 @@ func LogToDB(payload Log) bool {
 	// Handle possible error
 	if err != nil {
 		log.Fatal(err.Error())
+		tx.Rollback() //quit transaction if error
 		return false
 	}
 
-	// Close
-	defer rows.Close()
+	err = tx.Commit() //finish transaction
+	if err != nil {
+		log.Fatal(err.Error())
```

Comment:
No need for logging inside here, cuase error is a return value.
And don't use log.Fatal() cauase it will stop the whole application.